WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteCardAndItemTest.kt

```diff
@@ -32,6 +32,13 @@ class MySiteCardAndItemTest {
         assertThat(siteInfoCard.activeQuickStartItem).isTrue()
     }
 
+    @Test
+    fun `given site info block is active when focus point on subtitle`() {
```

Comment:
Nitpick: I think we can either remove `given` from the test description or convert it to a proper `given-when-then` format.

Reply:
@ashiagr Yup, I agree. Don't know how I missed that. Have changed the test description to a proper `given-when-then` in [430e39](https://github.com/wordpress-mobile/WordPress-Android/pull/16373/commits/d430e3947089f4c2d57d0ce7091a7202452e5cb5)